specification/cognitiveservices/data-plane/LUIS/Runtime/v2.0/LUIS-Runtime.json

```diff
@@ -0,0 +1,338 @@
+        ],
+        "x-ms-examples":{
+          "Successful Get Predictions From Endpoint Via Post request": {
+            "$ref": "./examples/prediction/SuccessfulGetPredictionsFromEndpointViaPostRequest.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Q": {
+      "type": "string",
+      "example": "\"forward to frank 30 dollars through HSBC\""
+    },
+    "LuisResult": {
+      "type": "object",
+      "properties": {
+        "query": {
+            "type": "string",
+            "x-nullable": true
+        },
+        "topScoringIntent": {
+          "$ref": "#/definitions/IntentModel",
+          "x-nullable": true
+        },
+        "intents": {
+          "type": "array",
+          "x-nullable": true,
+          "items": {
+            "$ref": "#/definitions/IntentModel"
+          }
+        },
+        "entities": {
+          "type": "array",
+          "x-nullable": true,
+          "items": {
+            "$ref": "#/definitions/EntityModel"
+          }
+        },
+        "compositeEntities": {
+          "type": "array",
+          "x-nullable": true,
+          "description": "The composite entities extracted from the utterance.",
+          "items": {
+            "$ref": "#/definitions/CompositeEntityModel"
+          }
+        }
+      }
+    },
+    "CompositeEntityModel": {
+      "description": "LUIS Composite Entity.",
+      "type": "object",
+      "properties": {
+        "parentType": {
+          "type": "string",
+          "description": "Type/name of parent entity."
+        },
+        "value": {
+          "type": "string",
+          "description": "Value for composite entity extracted by LUIS."
+        },
+        "children": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CompositeChildModel"
+          }
+        }
+      },
+      "required": [
+        "parentType",
+        "value",
+        "children"
+      ]
+    },
+    "CompositeChildModel": {
+      "description": "Child entity in LUIS composite entity.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Type of child entity."
+        },
+        "value": {
+          "type": "string",
+          "description": "Value extracted by LUIS."
+        }
+      },
+      "required": [
+        "type",
+        "value"
+      ]
+    },
+    "IntentModel": {
+      "description": "Intent returned from LUIS.",
+      "type": "object",
+      "properties": {
+        "intent": {
+          "type": "string"
+        },
+        "score": {
+            "type": "number"
+        }
+      }
+    },
+    "EntityModel": {
+      "description": "Entity returned from LUIS.",
+      "type": "object",
+      "properties":{
+        "entity": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "startIndex": {
+          "type": "number"
```

Comment:
It would be a good idea to have `maximum` and `minimum` values for numbers if you can.

Reply:
Added description with their meaning. These values change based on the input. They represent a match within the input query